test/config.js

```diff
@@ -10,6 +10,12 @@ describe('config', function () {
       assert.equal(app.get('foo'), 'bar');
```

Comment:
The changes to this file are lifted from 2ddae300851a6f5e4626d77ef50822146d17d06f.